apps/react/src/app/components/ToggleMenu/ToggleMenu.tsx

```diff
@@ -0,0 +1,49 @@
+import { Button, Paper, Slide } from '@mui/material';
+import { Box } from '@mui/system';
+import { FC, memo, ReactElement, useState } from 'react';
+import KeyboardArrowDownIcon from '@mui/icons-material/KeyboardArrowDown';
+import KeyboardArrowUpIcon from '@mui/icons-material/KeyboardArrowUp';
+
+import './ToggleMenu.css';
+
+interface Props {
+
+  /** Menu content. */
+  readonly children: ReactElement;
+}
+
+const ToggleMenuComponent: FC<Props> = ({ children }: Props) => {
```

Comment:
You can omit annotation when using generic.
```suggestion
const ToggleMenuComponent: FC<Props> = ({ children }) => {
```